### PR TITLE
Upgrade trace to decrease GC pressure.

### PR DESCRIPTION
**Pull request by mask-pp**

### CODE REVIEW

**Review by 0xmountaintop — DISMISSED** [review]

**Comment by lispc** [comment]
So we can merge this?

**Comment by 0xmountaintop** [comment]
> So we can merge this? 

no i think this can be improved

**Comment by lispc** [comment]
> > So we can merge this?
> 
> no i think this can be improved

Ok

File: core/vm/logger_trace.go [hunk 1]
```diff
@@ -126,14 +126,15 @@ func getWrappedProofForAddr(l *StructLogger, address common.Address) (*types.Acc
...
-		CodeHash: l.env.StateDB.GetCodeHash(address),
-		Proof:    encodeProof(proof),
-	}, nil
+	accountProof := types.NewAccountProofWrapper(
+		address,
+		l.env.StateDB.GetNonce(address),
+		(*hexutil.Big)(l.env.StateDB.GetBalance(address)),
+		l.env.StateDB.GetCodeHash(address),
+	)
+	encodeProof(&accountProof.Proof, proof)
```
Comment (0xmountaintop) [inline core/vm/logger_trace.go]: can we include  `Proof` and `Storage` into `NewAccountProofWrapper`?

It makes little sense that we "new" a `AccountProofWrapper` but set `Proof` and  `Storage` later

**Review by lispc — APPROVED** [review]

**Review by 0xmountaintop — APPROVED** [review]

**Pull request merged**